tests/Features/Macro.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Illuminate\Support\Traits\Macroable;
+use PHPUnit\Framework\TestCase;
+
+uses(Macroable::class);
+
+it('can call chained macro method')->macro('bar', function () {
+    assertInstanceOf(TestCase::class, $this);
+
+    return $this;
```

Comment:
You don't need this return.

Reply:
https://github.com/pestphp/pest/pull/109/commits/bff97418edb5ca8b5493f24172780acbde2107b6